specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,11 +28,34 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2022-11
+tag: package-2023-05
 ```
 
 ---
 
+### Tag: package-2023-05
+
+These settings apply only when `--tag=package-2023-05` is specified on the command line.
+
+``` yaml $(tag) == 'package-2023-05'
+input-file:
+  - Microsoft.KubernetesConfiguration/stable/2023-05-01/extensions.json
+  - Microsoft.KubernetesConfiguration/stable/2023-05-01/fluxconfiguration.json
+  - Microsoft.KubernetesConfiguration/stable/2023-05-01/kubernetesconfiguration.json
+  - Microsoft.KubernetesConfiguration/stable/2023-05-01/operations.json
+```
+
+### Tag: package-preview-2023-05
+
+These settings apply only when `--tag=package-preview-2023-05` is specified on the command line.
+
+``` yaml $(tag) == 'package-preview-2023-05'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2023-05-01-preview/extensionTypes.json
+  - Microsoft.KubernetesConfiguration/stable/2023-05-01/operations.json
```

Comment:
Why does preview tag `package-preview-2023-05` contains file under stable folder `stable/2023-05-01`?

Reply:
> Why does preview tag `package-preview-2023-05` contain file under stable folder `stable/2023-05-01`?

@msyyc  We have two api versions in this PR `2023-05-01` and `2023-05-01-preview`

The file you are referring to is from the stable api-version 2023-05-01